source/_patterns/00-atoms/components/font-loader.mustache

```diff
@@ -0,0 +1,16 @@
+{{#fonts}}
+  <link rel="preload" href="{{#assetRewrite}}{{assetsPath}}{{{path}}}{{/assetRewrite}}" as="font">
+{{/fonts}}
+
+{{#fonts}}
+
+  <style>
+    @font-face {
+      {{#display}}font-display: {{display}};{{/display}}
```

Comment:
I don't think implements should have to know what these are, knowledge should remain here.